scripts/hotfixes/kdump/README.md

```diff
@@ -0,0 +1,48 @@
+# Kernel Dump Hotfix
+
+This directory includes the script and the library necessary for applying the hotfix to all reachable NCNs.
+
+## Usage
+
+1. Run `hotfix.sh`.
+
+    ```bash
+    ncn# /usr/share/doc/csm/scripts/hotfixes/kdump/hotfix.sh
+    ```
+
+   Example output:
+
+   ```text
+   Uploading hotfix files to ncn-m001:/srv/cray/scripts/common/ ... Done
+   Uploading hotfix files to ncn-m002:/srv/cray/scripts/common/ ... Done
+   Uploading hotfix files to ncn-m003:/srv/cray/scripts/common/ ... Done
+   Uploading hotfix files to ncn-s001:/srv/cray/scripts/common/ ... Done
+   Uploading hotfix files to ncn-s002:/srv/cray/scripts/common/ ... Done
+   Uploading hotfix files to ncn-s003:/srv/cray/scripts/common/ ... Done
+   Uploading hotfix files to ncn-s004:/srv/cray/scripts/common/ ... Done
+   Uploading hotfix files to ncn-w001:/srv/cray/scripts/common/ ... Done
+   Uploading hotfix files to ncn-w002:/srv/cray/scripts/common/ ... Done
+   Uploading hotfix files to ncn-w003:/srv/cray/scripts/common/ ... Done
+   Uploading hotfix files to ncn-w004:/srv/cray/scripts/common/ ... Done
+   Running updated create-kdump-artifacts.sh script on [11] NCNs ... Done
+   The following NCNs contain the kdump patch:
+   ncn-m001
+   ncn-m002
+   ncn-m003
+   ncn-s001
+   ncn-s002
+   ncn-s003
+   ncn-s004
+   ncn-w001
+   ncn-w002
+   ncn-w003
+   ncn-w004
+   This hotfix has completed.
+   ```
+
+## Background
+
+The original script and library used live in [metal-provision](https://github.com/Cray-HPE/metal-provision/blob/main/roles/ncn-common-setup/files/srv/cray/scripts/common/create-kdump-artifacts.sh)
+
+The only difference between the script in this hotfix and the script for CSM 1.3 should be the inclusion/exclusion of
```

Comment:
This seems like an odd thing to mention. Also doesn't really feel like "Background". I'd omit this part.

Reply:
Leaving it in.

Reply:
How is it helpful for customers to have this here? If it's here for the benefit of developers, it seems like the wrong place for it.

Reply:
Why does this matter so much? This is mainly emphasis that this script and the perm fix is the same, and for record keeping it shouldn't differ or its differences should be very explicit.

Reply:
is the issue with the clause mentioning the 3 differences?

Reply:
removed the clause.

Reply:
Yeah, I was talking here specifically about the 3 differences line. If just wanting to emphasize that this is essentially the same as the permanent fix, I'd say something closer to exactly that -- "This hot fix is essentially the same as the proper fix that is being introduced in CSM 1.3."

Reply:
Makes sense.